features/payment-handler.yml

```
@@ -0,0 +1,34 @@
+name: Payment Handler
+description: The Payment Handler API provides a standardized set of functionality for web applications to directly handle payments, rather than having to be redirected to a separate site for payment handling.
```

- "API" is generally to be avoided, according to the docs, but these APIs are typically referred to as such, so I'll leave it for Daniel to make the call in final review.
- remove `providing` and some passive verbing
- integrate a bit of the language from the spec, which i think was clearer than MDN
```suggestion
description: The Payment Handler API lets web applications handle requests for payments directly through the browser user interface, rather than redirecting users to a separate site for payment handling.
```

Hm, also might want to add another sentence which summarizes the components (a manager, some events) and how they interact in a usage scenario. Maybe pushing on length at that point though.